docs/rfcs/7-add-indentity-support.md

```diff
@@ -0,0 +1,93 @@
+- Author: Prnyself <281239768@qq.com>
+- Start Date: 2021-08-19
+- RFC PR: [beyondstorage/beyond-ctl#0](https://github.com/beyondstorage/beyond-ctl/pull/0)
+- Tracking Issue: [beyondstorage/beyond-ctl#0](https://github.com/beyondstorage/beyond-ctl/issues/0)
+
+# BCP-7: Add Identity Support
+
+## Background
+
+In [GSP-90], we introduced connection string to initialize a storager by string input.
+
+So we had originally planned not to introduce config file for beyond-ctl, for which, users can input the config of a
+storager in command line. For example:
+
+```
+beyondctl cp fs:///path/to/key s3://bucket_name/object_name
+```
+
+However, the connection may be too long for users to input every time, especially when it contains some params
+like `?endpoint=xxxx:xxx:xx`, or credential info which is usually sensitive.
+
+So we can write these identities' info into local config file, and support commands to `get`,
+`set`, `delete` identity.
+
+## Proposal
+
+So I propose to add identity support to record user's connection string with an alias in config file.
+
+### Config
+
+The config struct will be like:
+
+```go
+type Config struct {
+    Version string               `json:"version"`
```

Comment:
We will use json? How about  toml instead.

Reply:
What's the difference?

Reply:
`toml` is human-readable, easy to maintain, supports comments.

json is suitable for machines but not humans, we can add `--json` to support json output.